### DISCUSSION TITLE
Which is your favorite prefecture in Japan?

### DISCUSSION THREAD

**Discussion by Yuya274** [NAIST-SE/AttractingContributors#1146] [2024 Attraction Contributions]
Which is your favorite prefecture in Japan?

あなたの好きな都道府県について教えて下さい！

Ex：Hokkaido

例：北海道

https://github.com/Yuya274/Which-is-your-favorite-prefecture-in-Japan-.git

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git](https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git)

**Reply by 3207-Rhims** [reply]
https://github.com/3207-Rhims/AttractingContributors-Movie/            , can you contribute to my repo ? I just contributed in yours.

**Comment by ryosuke-sota** [comment]
I made a PR on your repo.
Could you contribute to mine? Here : https://github.com/ryosuke-sota/AttractingContributors.git

**Comment by anzhitingg** [comment]
Hello！
I have just made a PR on your repo. 
Could you contribute to mine? 
Here : https://github.com/anzhitingg/AttractingContributors-MBTI
Thank you very much！

**Comment by r-nishioka** [comment]
Hi. I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
[Where would you like to trip in Japan?](https://github.com/r-nishioka/AttractingContributors-Trip.git)

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Comment by Idescrew** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?

https://github.com/Idescrew/AttractingContributors
